### PR TITLE
paper: add desc+livecheck+depends_on

### PR DESCRIPTION
**Pull request by neutric**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/paper.rb [hunk 1]
```diff
@@ -4,8 +4,16 @@
 
   url "http://paperimg.meiyuan.in/pap.er_v#{version}.dmg"
   name "Paper"
+  desc "Live wallpaper app"
   homepage "https://paper.meiyuan.in/"
 
+  livecheck do
+    url :homepage
+    regex(/["'][^"']*pap\.er[._-]v?(\d+(?:\.\d+)+)\.dmg["']/i)
```
Comment (bevanjkay) [inline Casks/paper.rb:12]: @neutric I modified the regex slightly to be better anchored to the start and end of the area we are looking to capture. It also isn't necessary to use `strategy :page_match` as it is the default.

File: Casks/paper.rb [hunk 2]
```diff
@@ -4,8 +4,16 @@
...
+  end
+
+  depends_on macos: ">= :el_capitan"
```
Comment (bevanjkay) [inline Casks/paper.rb:15]: @neutric According to the `info.plist` file in `pa.per.app` it should support back to Yosemite. Where did you find information about the minimum version?

```
	<key>LSMinimumSystemVersion</key>
	<string>10.10</string>
```

Comment (neutric) [reply Casks/paper.rb:15]: The homepage says "macOS X 10.11 or higher version" at the very bottom.

**Review by bevanjkay — APPROVED** [review]

File: Casks/paper.rb (hunk 1 above)
Comment (neutric) [reply Casks/paper.rb:12]: @bevanjkay Thanks for the modification! Regarding ```strategy :page_match```: the documentation on the [livecheck](https://docs.brew.sh/Cask-Cookbook#stanza-livecheck) stanza might need some rewording then. It currently says 

> a livecheck should specify which strategy should be used to extract the version

whereas the documentation on [brew livecheck](https://docs.brew.sh/Brew-Livecheck) says
> Only use strategy when it’s necessary.

**Pull request merged**